drizzlepac/alignimages.py

```diff
@@ -267,17 +262,14 @@ def run_align(input_list, archive=False, clobber=False, debug=False, update_hdr_
         Table which contains processing information and alignment results for every raw image evaluated
 
     """
-    log.info("*** HLAPIPELINE Processing Version {!s} ({!s}) started at: {!s} ***\n".format(__version__,
-                                                                                            __version_date__,
-                                                                                            util._ptime()[0]))
+    log.info("*** HLAPIPELINE Processing Version {!s} ({!s}) started at: {!s} ***\n".format(__version__, __version_date__, util._ptime()[0]))
 
     # Define astrometric catalog list in priority order
     catalog_list = ['GAIADR2', 'GAIADR1']
 
     # 0: print git info
     if print_git_info:
-        log.info(
-            "{} STEP 0: Display Git revision info {}".format("-" * 20, "-" * 48))
+        log.info("-------------------- STEP 0: Display Git revision info  ------------------------------------------------")
```

Comment:
Looks like this is the old way to display the divider line which violates PEP8 line length.  Keep Mike's new and improved way.